cmd/podman/main.go

```diff
@@ -92,6 +108,8 @@ func init() {
 	rootCmd.PersistentFlags().BoolVar(&MainGlobalOpts.Syslog, "syslog", false, "Output logging information to syslog as well as the console")
 
 	rootCmd.PersistentFlags().StringVar(&MainGlobalOpts.TmpDir, "tmpdir", "", "Path to the tmp directory")
+	rootCmd.AddCommand(rootCommands...)
```

Comment:
So how are these different?

Reply:
If one of these is remote and one is not, can we document that somehow? Because I have no idea how to tell which is which